src/test/java/com/wikia/webdriver/common/dataprovider/TrackingOptInDataProvider.java

```diff
@@ -6,16 +6,15 @@
 
 public class TrackingOptInDataProvider {
 
-  private static final String
-      ADS_KIKIMORA_PATTERN =
-      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*";
+  private static final String ADS_KIKIMORA_OPT_IN_ACCEPTED_PATTERN =
+      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*u=0.*";
+  private static final String ADS_KIKIMORA_OPT_IN_REJECTED_PATTERN =
+      "https?://.*beacon\\.wikia-services\\.com/__track/special/adeng.*u=-1.*";
   private static final String ADS_KRUX_PATTERN = "https?://.*cdn\\.krxd\\.net.*";
-  private static final String
-      ADS_NETZ_ATHLETEN_PATTERN =
+  private static final String ADS_NETZ_ATHLETEN_PATTERN =
       "http?://.*adadapter\\.netzathleten-media\\.de/.*/naMediaAd\\.js.*";
   private static final String ADS_MOAT_PATTERN = "https?://.*pixel\\.gif.*";
```

Comment:
This pattern will match any domain not only `moatads.com`. Therefore, I suggest to rename it or make more moat-oriented ;)